electionleaflets/settings/base.py

```diff
@@ -54,8 +101,7 @@
 # Examples: "http://foo.com/media/", "/media/".
 ADMIN_MEDIA_PREFIX = '/admin_media/'
 
-# Don't share this with anybody.
-SECRET_KEY = "elyfryi8on!dmw&8b3j-g0yve4u&%4_6%(tf3*)@#&mq*$yzhf^6"
+SECRET_KEY = env('SECRET_KEY', default='INSECURE')
```

Comment:
Ref above comment, so I expect this line will change anyway, but it's good to have quite a long "placeholder" key here, as when it's too short hash collisions are more common. I've had this problem in the past, and it's _not_ fun to debug! :)